pkg/transfers/inbound/cleanup.go

```diff
@@ -25,13 +28,34 @@ func Cleanup(logger log.Logger, agent upload.Agent, dl *downloadedFiles) error {
 	if err := deleteFilesOnRemote(logger, agent, dl.dir, agent.ReturnPath()); err != nil {
 		el.Add(err)
 	}
+	if el.Empty() {
+		return nil
+	}
+	return el
+}
+
+// CleanupEmptyFiles deletes empty ACH files if file is older than value in config
+func CleanupEmptyFiles(logger log.Logger, agent upload.Agent, dl *downloadedFiles, now time.Time, after time.Duration) error {
+	var el base.ErrorList
+
+	if after <= 0*time.Second {
+		logger.Logf("deleting empty file requires after > 0. currently: %s", after)
+		return nil
+	}
 
+	if err := deleteEmptyFiles(logger, agent, dl.dir, agent.InboundPath(), now, after); err != nil {
+		el.Add(err)
+	}
+	if err := deleteEmptyFiles(logger, agent, dl.dir, agent.ReturnPath(), now, after); err != nil {
+		el.Add(err)
+	}
```

Comment:
Up to you if you want to add this, but you could simplify to:
```
for _, path := range []string{agent.InboundPath(), agent.ReturnPath()} {
  if err := deleteEmptyFiles(logger, agent, dl.dir, path, now, after); err != nil {
		el.Add(err)
	}
}
```

Reply:
I like this a lot better. Pushed

Reply:
We should be consistent with `Cleanup` above. It's confusing when different coding styles are used within a file.